content/docs/operating/troubleshooting.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Troubleshooting
+sort_rank: 2
+---
+
+# Troubleshooting
+
+## Device Errors
+
+### Device is not showing up in Cluster/Talaria
+
+- Problem: Parados has the incorrect url.
+
+  - Solution: Validate Parados has the correct url.
+
+    ```bash
+    docker run -e URL=http://HOSTNAME:PRIMARY_PORT rdkb-simulator
+    # HOSTNAME should be either a talaria or petasos address, with
+    # the correct corresponding PRIMARY_PORT
+    ```
+
+- Problem: TLS is inconsistent between talaria and petasos.
+
+  - Fixed Solution: Change talaria and petasos Configuration to be consistent with `http` or `https`.
+
+    ```yaml
+    # talaria  
+    service:
+      defaultScheme: http
+      fixed:
+        - http://talaria:6200
```

Comment:
This example has fixed values but the consul one has variables.  Seems like we're moving towards variables, so maybe add them here?